docs/acceptance/db.md

```diff
@@ -0,0 +1,65 @@
+## Database Deliverable Acceptance Test 
+
+### Description of Deliverable
+
+This deliverable includes the implementation of an ETL process, the design of a database schema, the selection of a format for raw data storage, infrastructure as code, database configuration, and accompanying documentation.
+
+The acceptance test below provides steps to verify that the deliverable meets our agreed-upon specifications.
+
+### Input Data
+
+The input data is stored in Parquet format on AWS S3 (object storage), specifically in the file `space2stats_updated.parquet`. Any additional fields must be appended to this file. The Parquet file is tabular with the following columns:
+- `hex_id`
+- `{aggregation_method[sum, mean, etc.]}_{variable_name}_{year}`
+
+### Database Setup
+
+You can use a local database for this acceptance test by running the following command in the root directory:
+
+```bash
+docker-compose up
+```
+
+Alternatively, you can connect to a remote database, such as the Tembo database used for production.
+
+### Data Ingestion
+
+Set the database environment variables in `db.env`:
+
+```bash
+DB_HOST=localhost
+DB_PORT=5432
+DB_NAME=postgis
+DB_USER=postgres
+DB_PASSWORD=password
+DB_TABLE_NAME=space2stats
+```
+
+> Note: If using the `docker-compose` approach, the above configuration is suitable.
+
+To ingest data, run the following script:
+
+```bash
```

Comment:
is the chunk parquet step still necessary or can we delete those scripts? 

`download_parquet.sh` creates space2stats.parquet but `load_to_prod.sh` expects space2stats_updated.parquet

Reply:
Good catch! Removed chunking scripts and renamed file to `space2stats.parquet` in `load_to_prod.sh`.